continuous-integration/starter-kits/README.md

```diff
@@ -1,6 +1,97 @@
 # Starter Kits
 
-This page contains starter kit information, which represent templates, code and configuration to help you get started quickly with continuous integration best practices described in this overall guide. Please see categories and links below for details. 
+This page contains starter kit information, which represent templates, code and configuration to help you get started quickly with continuous integration (CI) best practices described in this overall guide. Please see categories and links below for details. 
+
+## Build, Release and Publish Automation
+This section contains links to sample actions, templates and configurations that demonstrate [CI systems](../reference-architectures/) in practice on various platforms using different development languages.
+
+### Python Starter Kit
+A complete, deployment-ready Python 3 application that's bundled into a reusable template repository for quick implementation.
+
+#### Integrated Build, Release and Publish on GitHub
+This product demonstrates a GitHub-based strategy of [end-to-end CI automation](../reference-architectures/) using modern Python tooling to demonstrate publishing on release.
+* Build automation using [GitHub](https://github.com/), [Docker](https://www.docker.com/), [GitHub Actions](https://github.com/features/actions) and [PyPi](https://pypi.org/)
+* Compile, tag and release, and publish to the [Python Package Index (PyPi)](https://pypi.org/)
+* Modern [PEP 517](https://peps.python.org/pep-0517/)/[518](https://peps.python.org/pep-0518/) compliant tooling using [declarative](https://en.wikipedia.org/wiki/Declarative_programming) [TOML](https://toml.io/en/)-based configurations inside a `pyproject.toml` file.
+
+Starter Kit:
+- [SLIM Starterkit Python](https://github.com/NASA-AMMOS/slim-starterkit-python)
+
+To leverage this template, make sure to do the following:
+1. Discuss with your development team continuous integration best practices and seek consensus on a workflow to build, publish and release software. 
+2. Create a new repository by:
+   a. [Creating a new repository using our template repository](https://github.com/NASA-AMMOS/slim-starterkit-python/generate) (GitHub only)
+   b. [Cloning and manually using our template repository](https://github.com/NASA-AMMOS/slim-starterkit-python). 
+> **Requirement**  
+    • Admin rights are necessary to set up `GitHub Secrets`.  
+    • Name tokens identically in both Test PyPi and PyPi. We'll substitute values later using a [sandbox strategy](http://agiledata.org/essays/sandboxes.html) to test.
+3. Setup account credentials:
+   1. [Test PyPi](https://test.pypi.org/account/register/) and [PyPi](https://pypi.org/account/register/) 
```

Comment:
It would be good to break out Test PyPi and PyPi into separate steps entirely.

Reply:
Change made.